pom.xml

```diff
@@ -312,16 +312,14 @@ Import-Package: \\
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
-          <version>2.10.3</version>
+          <version>3.2.0</version>
           <configuration>
             <failOnError>!${quality.skip}</failOnError>
             <!-- Turn off doclint -->
-            <additionalparam>-Xdoclint:none</additionalparam>
-            <!-- Frames are deprecated and will disappear at some point -->
-            <additionalparam>--frames</additionalparam>
+            <doclint>none</doclint>
             <!-- Prevents the "undefined" module in path name, which causes a file not found when searching -->
             <additionalJOption>--no-module-directories</additionalJOption>
```

Comment:
It may fix the searching, but is causes another issue, i.e. all links to Oracle no longer work for standard Java classes such as Object, String etc.

Reply:
Yes, at the time it was the lesser of two evils. I see you found a better workaround. Excellent!